game-of-life/README.md

```diff
@@ -1,17 +1,31 @@
-Write a cellular automata that models the [Game of Life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life)
+# Game of Life
 
-Requirement Cards:
+## Background 
 
-1. A cell can be made "alive"
+The Game of Life is a cellular automaton. 
 
-2. A cell can be "killed"
+It it discussed in detail on [Wikipedia](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life).
 
-3. A cell with fewer than two live neighbours dies of under-population
+## Rules
 
-4. A cell with 2 or 3 live neighbours lives on to the next generation
+The game of life is played on a two dimensional grid.
 
-5. A cell with more than 3 live neighbours dies of overcrowding
+At each turn, the state of a cell may either be "alive" or "dead"; determined by the following rules:
 
-6. An empty cell with exactly 3 live neighbours "comes to life"
+1. A cell with fewer than two live neighbours dies of under-population
+2. A cell with 2 or 3 live neighbours lives on to the next generation
+3. A cell with more than 3 live neighbours dies of overcrowding
+4. An empty cell with exactly 3 live neighbours "comes to life"
+
+## Tasks
+
+1. Implement a grid for the game to take place on. Carefully consider what happens at the edges.
```

Comment:
I'm not sure whether 'Carefully consider what happens at the edges.' should be in 1. or 2 - it might be good to get people to write a bit of code before hitting them with scary edges stuff?

Reply:
I'd agree with that – so often in a pairing test, getting the first few tasks completed is a good confidence boost. Having said that, I don't know how easy it would be to retrofit this in an implementation if you hadn't added wrapping in 2. Moving it to 2. might mitigate the scariness somewhat.

Perhaps we could add the wrapping requirement on L11, `The game of life is played on a two dimensional grid. The grid wraps around, so a cell on the far right hand edge is considered to be 'next to' a cell at the same Y co-ordinates on the far left.`

Reading that back, I'm not sure it's better!

Reply:
I thought about this, and think the decision making will be interesting here.

I'm not actually sure that it benefits that much from spherical geometry, the assumption is "infinite size" and I think (candidates please don't read this bit ;) ) I'd be happy with any well reasoned approach.

Reply:
Is this qualifier too leading? Could it be considered and edge case and thus allow the candidate to think about this themselves and we prompt them about it before moving on if they haven't? This would be a similar approach to Scrabble, for example, and bad input to the score word function in part 1.

Reply:
Slightly perhaps, but I want to make sure that the candidate and interviewer are 100% on the same page about how they're modelling this- as it's a really easy thing to cross wires about.

Reply:
👍🏽  to the intent.

I think I agree with @jonathonherbert 's suggestion to have the qualifier with the "rules" section above.

Reply:
```suggestion
1. Implement a grid for the game to take place on. 
2. Carefully consider what happens at the edges.
3. Implement the game of life for a single iteration.
4. Display the results for a single iteration.
5. Animate the game so that it is played in real time.
```

I'd be happiest with @philmcmahon's suggestion here. I don't feel that this belongs in the Rules, as it isn't a rule of the gol. 

The point of these task descriptions is something for an interviewer and candidate to work through together to assess the candidate. The previous wording was unclear, and any problems with that make it much harder to make a fair assessment. 

Also, I'm afraid this is quite a lot of scrutiny over a small aspect of the change; which I fear isn't likely to be exceptionally beneficial.